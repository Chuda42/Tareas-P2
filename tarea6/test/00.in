# Implementar structs, las operaciones de crear y liberar, e inicioCadena 
Fin
