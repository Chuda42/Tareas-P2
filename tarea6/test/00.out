1>#  Implementar structs, las operaciones de crear y liberar, e inicioCadena .
2>Fin.
